### ISSUE TITLE
Support VR180 playback on YouTube

### ISSUE THREAD

**Issue by cvan** [MozillaReality/FirefoxReality#793]
## Hardware
All

## Steps to Reproduce

1. Load a VR180 video (not a 360 video) on YouTube https://youtube.com/playlist?list=PLU8wpH_Lfhmu_kk955BFwvV0yT0PEcenA
2. Notice the unwrapped video is not shown (despite the new UA Override).

## Current Behavior

Notice the unwrapped video is not shown (despite the new UA Override).

With the new UA Override, properly unwrapped YouTube 360 videos are shown with the new media controls (#330).

VR180 is a kind-of proprietary format created by YouTube/Google.

If you dig into the source code of a video detail page, you can find the original unwrapped video track. The browser *could* do some trickery to extract that info. from a YouTube VR180 video page, but we probably should not start that precedent. That is a job for WebExtensions' content scripts.

## Expected Behavior
For the user to be able to use FxR's media controls for setting the Projection Mode, YouTube should serve unwrapped 180 videos for the same UA we are using to get unwrapped 360 videos.

## Possible Solution

Ideally, we reach out to YouTube to ask them to change the VR180 video pages to serve a similar video player as the 360 ones.

Or the browser can introduce functionality to parse VR180 metadata and streamlined video playback.

## Context

Oculus Browser also does not support VR180.

The native Oculus Go app for YouTube plays VR 180 extremely well. The VR180 catalogue is relatively small today but seems to be growing rapidly.

For any solution we chose, we should move quickly so FxR is ready when VR180 becomes commonplace (even if for only cinephiles and today's VR enthusiasts).

## Error Logs and Stack Traces

<details open>

```
```

</details>

**Comment by bluemarvin** [comment]
Is this still an issue? @daron-walters or @MortimerGoro is this something we can address with the Web Extension? We should close this if no longer applicable.

**Comment by MortimerGoro** [comment]
The projection is broken. The problem is that we don't get the raw equirectangular video from Youtube. So we have to research a way to get it, as we did with 360 videos (Faking the UA). It will require some research. It might not be easy to guess, last time I checked Oculus Browser have the same problem. So Youtube hasn't checked the "Oculus UA" with 180VR format.

It might be a intended behavior from Youtube, so they can force users to use the Youtube App instead of browsers
